packages/server/src/fhir/lookups/address.ts

```diff
@@ -53,15 +53,21 @@ export class AddressTable extends LookupTable<Address> {
    *   address            | address
    *   address-city       | city
    *   address-country    | country
-   *   address-postalcode | postalcode
+   *   address-postalcode | postalCode
```

Comment:
This bug has apparently been here forever.  Currently a search for `Patient?address-postalcode` triggers a 500.